common/types/mapper/proto/history.go

```diff
@@ -249,6 +249,44 @@ func ToHistoryGetDLQReplicationMessagesResponse(t *historyv1.GetDLQReplicationMe
 	}
 }
 
+func FromHistoryGetFailoverInfoRequest(t *types.GetFailoverInfoRequest) *historyv1.GetFailoverInfoRequest {
```

Comment:
nit: shall we add some tests in `history_test.go`?

Reply:
Add unit tests